common/src/main/java/com/microsoft/identity/common/internal/platform/Device.java

```diff
@@ -64,4 +71,25 @@ public static synchronized IDevicePopManager getDevicePoPManagerInstance() throw
         }
     }
 
+    /**
+     * Determines if requests made on this device can successfully use AT/PoP APIs.
+     * @param context The current application's {@link Context}.
+     * @return True if AT/PoP requests can be made from this device. False otherwise.
+     */
+    public static boolean isDevicePoPSupported(@NonNull final Context context) {
```

Comment:
Code marker?

Reply:
For me - instructions on doing this here:
https://github.com/AzureAD/microsoft-authentication-library-common-for-android/blob/paul/update-perf-testing-documentation/uiautomationutilities/perf_tool/readme.md#steps-to-add-a-new-marker-and-see-it-in-the-report

Reply:
Took a stab at this here https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/1456/commits/5fb8270e059caf42c42bf8a2f7c3cd3557876667

@paulkagiri This gave me a chance to dogfood your instructions (which I [provided feedback on here](https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/1455)!) Can you review my changes and see if I followed your instructions correctly?